src/alire/alire-toml_index.adb

```diff
@@ -1860,7 +1860,11 @@ package body Alire.TOML_Index is
                  (Project            => +(+Pkg.Name),
                   Version            => R.Version,
                   Origin             => Origin,
-                  Notes              => +Pkg.Common.Notes,
+                  Notes              => +US.Head
+                    (Pkg.Common.Notes, Alire.Max_Description_Length),
+                  -- It crops too long notes, so something TODO about this
+                  -- Since it didn't fail before, I guess they weren't added
+                  --  this way (only as property?)
```

Comment:
Exactly :-) Given that there are more powerful Notes properties, what is the purpose of this argument? Could we merge the two?

Reply:
IIRC, in the previous index, Note was not a property but a unique field of the release. Comments OTOH where managed as properties (and there could be several in the same release). 

I see in line 1497 that you are importing the note as a comment. This is conflating the two, so when a release is later exported it will have been modified by alr, which to me is a no-no.

I propose, for now, to leave notes as they were originally (I can fix that in this same PR) and, if we want to homogenize the notes/comments situation, to open a specific issue for it. It's certainly confusing having notes and comments; perhaps we can change to general(one line descr + long desc) + release(notes) and that's it? But we can discuss that in the issue, if you agree.

Reply:
Thanks for the clarification and for opening #92! I agree we should do this as a next step.